### PR TITLE
document console API tutorial

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/developer-experience/issues/142.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
Alright @pepopowitz -- I could litter this with comments and questions, but want you to have a fresh look first.

Feel free to make as many nitpicky critiques as you'd like, particularly based on how much code we want to offer up for new users.

Once complete, I can backport this. Once merged, we can get started on Optimize?

**Comment by pepopowitz** [comment]
@christinaausley I'll review this today but it will take me a bit of time -- I'm going to approach it as if I'm new to it, and walk through it from scratch.

**Comment by christinaausley** [comment]
> @christinaausley I'll review this today but it will take me a bit of time -- I'm going to approach it as if I'm new to it, and walk through it from scratch.

No rush here 👍

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:109]: Maybe "Edit the `addClient` function" instead? The current phrasing makes me think that I'm looking at a function that I've edited in a previous step.

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:141]: blocking: I think we should include all of the code in these steps, rather than just the difference between this function and the previous. I think there are too many opportunities for them to get something wrong if we don't.

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:172]: blocking: Similar comment as the previous function, about showing them _all_ the code here instead of only the differences.

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:198]: blocking: Similar comment as the previous function, about showing them _all_ the code here instead of only the differences.

**Review by pepopowitz — CHANGES_REQUESTED** [review]
This is awesome, thanks for writing this up!

I have lots of comments! Most of them are non-blocking. The only ones I feel strongly enough about are labelled `blocking: `.

**Comment by pepopowitz** [comment]
Oh, one other thing I realized we talked about doing, but didn't yet --- we're going to put the code we wrote for the tutorial in a `completed` folder, so they can see what things should look like when they're done. 

I will move things around in the code example repo! And I'll leave it up to you to figure out a place to mention that the `completed` folder exists, if you think we should.

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:58]: There are some indentation discrepancies here and in other code blocks -- I think a result of the fact that in the code repo, they're indented another level, but here we're left-aligning them either intentionally or because of prettier. Which do you think we should align to -- zero indentation, or what is in the code repo?

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:159]: ❤️ I love this re-write -- includes all lines of code, but breezes through the duplicates a little quicker than the first time, and highlights the changes. 💯

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:110]:
```suggestion
1. Edit the `addClient` function, incorporate the access token, and add your settings in the `.env` file. Note that this function destructures the `clientName` as the first item in an array passed in.
```

I'm not sure this is the right place to call it out, but this is a noteworthy difference between this function and the previous.

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:263]:
```suggestion
## If you get stuck

Having trouble configuring your API calls or want to examine an example of the completed tutorial? Navigate to the `completed` folder in the [GitHub repository](https://github.com/camunda/camunda-api-tutorials/tree/main/completed), where you can view an example `administration.js` file.

## Next steps

You can script several additional API calls as outlined in the [Administration API reference material](/apis-tools/administration-api/administration-api-reference.md).
```

What do you think about separating these paragraphs into different sections? The `completed` folder is more of a hint for when they get stuck _on this tutorial_, not something they would try next.

**Review by pepopowitz — DISMISSED** [review]
None of my feedback is blocking! 

This is amazing, THANK YOU!

File: docs/apis-tools/console-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/console-api/tutorial.md:4]: Oh, I just realized! These should probably say `administration-api` instead of `console-api`....and the file path should change...and there are a couple places in the content where we say "Console API" instead of "Administration API".

File: docs/apis-tools/console-api/tutorial.md
Comment (christinaausley) [reply docs/apis-tools/console-api/tutorial.md:58]: Let's go with the code repo 👍 I'll adjust.

File: docs/apis-tools/administration-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/administration-api/tutorial.md:113]:
```suggestion
async function addClient([clientName]) {
```

Missing an open curly brace for the function definition.

File: docs/apis-tools/administration-api/tutorial.md
Comment (pepopowitz) [inline docs/apis-tools/administration-api/tutorial.md:170]:
```suggestion
async function viewClient([clientId]) {
```

Missing an open curly brace for the function definition.

File: versioned_docs/version-8.4/apis-tools/administration-api/tutorial.md
Comment (pepopowitz) [inline versioned_docs/version-8.4/apis-tools/administration-api/tutorial.md:113]:
```suggestion
async function addClient([clientName]) {
```

Missing an open curly brace for the function definition.

File: versioned_docs/version-8.4/apis-tools/administration-api/tutorial.md
Comment (pepopowitz) [inline versioned_docs/version-8.4/apis-tools/administration-api/tutorial.md:170]:
```suggestion
async function viewClient([clientId]) {
```

Missing an open curly brace for the function definition.

**Review by pepopowitz — COMMENTED** [review]
One last tiny thing I noticed! Missing curly braces on the function definitions. I'll commit them through the GitHub UI as soon as this review is posted.

**Review by pepopowitz — APPROVED** [review]
Ship this, it's beautiful!!!! 💪 :shipit: 🚢

**Comment by christinaausley** [comment]
We can announce this in the release presentation next week, but I'll share in Console's Slack and follow up with the Optimize team after this is actually live in the docs. We can talk Optimize next steps next week! 🥳

**Pull request merged**
